extension-manifest-v3/src/pages/panel/views/home.js

```diff
@@ -106,168 +75,137 @@ export default {
-              </ui-text>
-              <ui-text type="body-m" layout="block:center width:::245px">
-                Navigate to a website to see Ghostery in action.
-              </ui-text>
-            </div>
-          `}
-          ${store.ready(stats) &&
-          html`
-            ${options.terms &&
-            options.paused &&
+        ${options.terms
+          ? stats.domain &&
             html`
               <gh-panel-pause
                 onaction="${togglePause}"
                 paused="${options.paused.find(({ id }) => id === stats.domain)}"
               ></gh-panel-pause>
-            `}
-            <ui-panel-stats
-              domain="${stats.domain}"
-              categories="${stats.categories}"
-              trackers="${stats.trackers}"
-              dialog="${TrackerDetails}"
-              type="${options.panel.statsType}"
-              layout="margin:2"
-              ontypechange="${setStatsType}"
-            >
-            </ui-panel-stats>
-          `}
-          ${store.ready(stats) &&
-          html`
-            <gh-panel-feedback
-              modified=${stats.trackersModified.length}
-              blocked=${stats.trackersBlocked.length}
-            ></gh-panel-feedback>
-          `}
-
-          <gh-panel-options>
-            <span slot="header">Ghostery settings</span>
-            <ui-text color="gray-900">
-              <a
-                href="${options.terms ? SETTINGS_URL : ONBOARDING_URL}"
-                onclick="${openTabWithUrl}"
-                layout="block"
-              >
-                <gh-panel-options-item
-                  icon="ads"
-                  enabled="${options.blockAds}"
-                  terms="${options.terms}"
-                >
-                  Ad-Blocking
-                </gh-panel-options-item>
-                <gh-panel-options-item
-                  icon="tracking"
-                  enabled="${options.blockTrackers}"
-                  terms="${options.terms}"
+            `
+          : html`
+              <gh-panel-button>
+                <a
+                  href="${chrome.runtime.getURL(
+                    '/pages/onboarding/index.html',
+                  )}"
+                  layout="row center gap:0.5"
+                  onclick="${openTabWithUrl}"
                 >
-                  Anti-Tracking
-                </gh-panel-options-item>
-                <gh-panel-options-item
-                  icon="autoconsent"
-                  enabled="${options.blockAnnoyances}"
-                  terms="${options.terms}"
+                  <ui-icon name="pause"></ui-icon>
+                  Enable Ghostery
+                </a>
+              </gh-panel-button>
+            `}
+        <gh-panel-container>
+          ${stats.domain
+            ? html`
+                <ui-panel-stats
+                  domain="${stats.domain}"
+                  categories="${stats.topCategories}"
```

Comment:
white space?

Reply:
Not sure what you mean, this is just prettier...